dist/basement.css

```diff
@@ -34,7 +34,9 @@
 
 .text-overflow-clip { text-overflow: clip; }
 
-.underline { text-decoration: underline; }
```

Comment:
hey let's keep this in for now to avoid breaking changes until we do a 2.0 release?

Reply:
fair!